partcad/src/partcad/part_factory_scad.py

```
@@ -42,7 +61,59 @@ async def instantiate(self, part):
+                [
+                    wrapper_path,
+                    os.path.abspath(part.path),
+                    os.path.abspath(cwd),
+                ],
+                request_serialized,
+                session=self.session,
+            )
+            if len(errors) > 0:
+                error_lines = errors.split("\n")
+                for error_line in error_lines:
+                    part.error("%s: %s" % (part.name, error_line))
```

Ruff (0.8.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)